lib/Charts/LineChart.js

```diff
@@ -260,18 +272,18 @@ function render(element, state) {
     }
 
     g.select('.x.axis')
-        .transition('1st').duration(transitionDuration)
+        .transition(t)
```

Comment:
I assume you used named transition to do concurrent transition, which no longer works this way in d4. The selection.transition method now takes an optional transition instance which can be used to synchronize a new transition with an existing transition: https://github.com/d3/d3/blob/master/CHANGES.md#transitions-d3-transition